src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -114,14 +114,15 @@
      *         The not-null name of the newly created game
      *
      * @throws IllegalArgumentException
-     *         if the specified name is null, empty or blank
+     *         if the specified name is null, empty, blank or longer than 128 characters
      *
      * @return A valid Activity instance with the provided name with {@link net.dv8tion.jda.api.entities.Activity.ActivityType#DEFAULT}
      */
     @Nonnull
     static Activity playing(@Nonnull String name)
     {
         Checks.notBlank(name, "Name");
+        Checks.check(name.length() <= 128, "Name must not be greater than 128 characters in length");
```

Comment:
Maybe we should trim the name first since discord will not care about leading or tailing spaces.

Reply:
Seems reasonable, maybe we should also move those checks into [this method](https://github.com/DV8FromTheWorld/JDA/blob/development/src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java#L124)  to avoid code duplication

Reply:
That would be problematic if we use that method later on for creating activities we receive. I prefer having this in the factory methods here instead.